blob/driver/driver.go

```diff
@@ -38,13 +38,20 @@ type Error interface {
 	BlobError() ErrorKind
 }
 
+// NoAs can be used as an As function that always returns false.
+var NoAs = func(interface{}) bool { return false }
```

Comment:
I'm not sure how useful it is to keep versus inlining, but if you're going to do it, just create a function instead of a variable.

Reply:
Yea, it was only used once. Removed.